### PR TITLE
Add ut for JindoRuntime with worker

### PR DESCRIPTION
**Pull request by frankleaf**
Signed-off-by: frankleaf <frank.wt@alibaba-inc.com>

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/jindo/const.go [hunk 1]
```diff
@@ -26,6 +26,9 @@ const (
 	JINDO_MASTER_PORT_MAX   = 11000
 	DEFAULT_WORKER_RPC_PORT = 6101
 	JINDO_WORKER_PORT_MAX   = 8100
+	PORT_NUM                = 2
```
Comment (allenhaozi) [inline pkg/ddc/jindo/const.go:29]: camel-case
[CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

File: pkg/ddc/jindo/worker.go [hunk 2]
```diff
@@ -214,3 +224,24 @@ func (e *JindoEngine) CheckWorkersReady() (ready bool, err error) {
 
 	return
 }
+
+// getWorkerSelectors gets the selector of the worker
+func (e *JindoEngine) getWorkerSelectors() string {
```
Comment (allenhaozi) [inline pkg/ddc/jindo/worker.go:229]: add new function for ut ? Will it be used elsewhere ?

File: pkg/ddc/jindo/worker.go [hunk 3]
```diff
@@ -214,3 +224,24 @@ func (e *JindoEngine) CheckWorkersReady() (ready bool, err error) {
...
+	}
+
+	selectorValue := ""
+	selector, err := metav1.LabelSelectorAsSelector(labelSelector)
+	if err != nil {
+		e.Log.Error(err, "Failed to parse the labelSelector of the runtime", "labels", labels)
+	} else {
+		selectorValue = selector.String()
+	}
+	return selectorValue
```
Comment (allenhaozi) [inline pkg/ddc/jindo/worker.go:246]: Why not just use `labels.Selector` instead of converting it to a string ?

File: pkg/ddc/jindo/const.go (hunk 1 above)
Comment (frankleaf) [reply pkg/ddc/jindo/const.go:29]: thx, but the consts seem not to be camel-case under project

File: pkg/ddc/jindo/worker.go (hunk 2 above)
Comment (frankleaf) [reply pkg/ddc/jindo/worker.go:229]: used by master internal, forget to update

File: pkg/ddc/jindo/worker.go (hunk 3 above)
Comment (frankleaf) [reply pkg/ddc/jindo/worker.go:246]: the selector get by LabelSelectorAsSelector, not the label's property

File: pkg/ddc/jindo/const.go (hunk 1 above)
Comment (allenhaozi) [reply pkg/ddc/jindo/const.go:29]: step-by-step modifications that already exist
but make sure the increments don't increase

Comment (frankleaf) [reply pkg/ddc/jindo/const.go:29]: thx，I'll do it all together next time if necessary

**Pull request merged**